supervision/detection/utils.py

```diff
@@ -592,6 +592,46 @@ def move_boxes(xyxy: np.ndarray, offset: np.ndarray) -> np.ndarray:
     return xyxy + np.hstack([offset, offset])
 
 
+def move_masks(
+    masks: np.ndarray, offset: np.ndarray, desired_shape: Optional[np.ndarray] = None
```

Comment:
Why:
- `desired_shape` is `Optional`?
- `desired_shape` is `np.ndarray` not tuple?
- `desired_shape` would have more elements than 2?

Reply:
I see I made some rushed mistakes.

1) If no shape is passed, the mask will be moved to the bottom-right corner with no padding. If we want to support unchanged `move_detections` signature, we should add `Optional` there too. Otherwise, it's better if we remove this one.
2) `tuple` makes more sense. I did some refactoring in my final commit and did not revert this.

3) I think, the parameter is called shape, so it makes sense to support passing `np_array.shape` here, which may have shape (height, width, channels) or (height, width).
* Do you think it's complicated? I could fix it to 2 dims. I figured, the pattern of [:2] on image shapes is something I've seen numerous times - seems trivial.

Reply:
I think it should be just `resolution_wh` required argument.